pdns/dnsdist-lua.cc

```diff
@@ -507,6 +526,57 @@ static void setupLuaConfig(LuaContext& luaCtx, bool client, bool configCheck)
                            config.mustResolve = boost::get<bool>(vars["mustResolve"]);
                          }
 
+                         if (vars.count("lazyHealthCheckSampleSize")) {
+                           auto value = std::stoi(boost::get<string>(vars.at("lazyHealthCheckSampleSize")));
+                           checkParameterBound("lazyHealthCheckSampleSize", value);
+                           config.d_lazyHealthChecksSampleSize = value;
+                         }
+
+                         if (vars.count("lazyHealthCheckMinSampleCount")) {
+                           auto value = std::stoi(boost::get<string>(vars.at("lazyHealthCheckMinSampleCount")));
+                           checkParameterBound("lazyHealthCheckMinSampleCount", value);
+                           config.d_lazyHealthChecksMinSampleCount = value;
+                         }
+
+                         if (vars.count("lazyHealthCheckThreshold")) {
+                           auto value = std::stoi(boost::get<string>(vars.at("lazyHealthCheckThreshold")));
+                           checkParameterBound("lazyHealthCheckThreshold", value, std::numeric_limits<uint8_t>::max());
+                           config.d_lazyHealthChecksThreshold = value;
```

Comment:
`lazyHealthCheckThreshold` being stored in `d_lazyHealthChecksThreshold` looks a bit weird (the extra s)